Popup.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="bootstrap-4.6.2-dist/css/bootstrap.min.css">
+    <link rel="stylesheet" href="Popup.css">
+    <title>Document</title>
+</head>
+<body>
+    <div id="pop" class="card">
+        <div class="pop">
+            <div class="image">
+                <img src="IMG/Copy-of-Copy-of-Recipe-Email-Image.png" style="width: 95%; margin: auto; padding: 10px;">
+            </div>
+            <div class="flex">
+                <div class="desc">
+                    <h3>Keep track of hundreds of components</h3><br>
+                    <div class="div">
+                        <button style="border-radius: 4px;">Codekit</button>
+                        <button style="border-radius: 4px;">Github</button>
+                        <button style="border-radius: 4px;">javascript</button>
+                        <button style="border-radius: 4px;">Bootstrap</button>
+                        <button style="border-radius: 4px;">Terminal</button>
+                        <button style="border-radius: 4px;">Codepen</button>
+                    </div>
+                </div>
+                <div class="buttons">
+                    <button>See live</button>
+                    <button>See source</button>
+                </div>
+            </div><br>
+            <p>Lorem ipsum dolor, sit amet consectetur adipisicing elit. Architecto animi, ullam quibusdam 
+                error pariatur reiciendis harum amet doloremque dolores iusto aperiam, laboriosam adipisci 
+                nobis deleniti recusandae quas esse ab illo!
+            </p>
+        </div>
+    </div>
+</body>
+</html>
```

Comment:
Create effort, so far

Please, when a user clicks on see project the above details should display on a pop-up modal screen not on a new page.
<img width="1020" alt="Screenshot 2025-05-20 at 5 18 39 AM" src="https://github.com/user-attachments/assets/3e4c3773-b653-494a-b9e9-5bf45b271331" />